app/src/androidTest/java/com/github/sdp/ratemyepfl/viewmodel/AddRoomAddReviewViewModelTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.Assert.*
 import org.junit.Rule
 import org.junit.Test
 
-class AddRoomReviewViewModelTest {
+class AddRoomAddReviewViewModelTest {
```

Comment:
Isn't there an extra 'Add' in the title?

Reply:
Yes, but if I am not mistaken, I deleted this file in one of the following commits. It shouldn't be in my branch anymore.

Reply:
Yes indeed, my bad